android/app/src/main/java/org/fossasia/openevent/fragments/DayScheduleFragment.java

```diff
@@ -47,16 +47,13 @@
 public class DayScheduleFragment extends BaseFragment implements SearchView.OnQueryTextListener {
 
     final private String SEARCH = "searchText";
-
+    @BindView(R.id.list_schedule)
+    RecyclerView dayRecyclerView;
+    @BindView(R.id.txt_no_schedule)
+    TextView noSchedule;
     private String searchText = "";
-
     private SearchView searchView;
-
     private SwipeRefreshLayout swipeRefreshLayout;
-
-    @BindView(R.id.list_schedule) RecyclerView dayRecyclerView;
-    @BindView(R.id.txt_no_schedule) TextView noSchedule;
-
```

Comment:
Please refrain from such refactoring in your code. It's impossible to review this way